client/src/__tests__/App.test.tsx

```diff
@@ -0,0 +1,16 @@
+import { act, render } from '@testing-library/react';
+import 'mobx-react-lite/batchingForReactDom';
+import React from 'react';
+import api from '~/api/test';
+import { App } from '~/components/App';
+import { StoreProvider } from '~/store';
```

Comment:
It would be much more convenient to have all this required things packed inside our custom testing render:
https://testing-library.com/docs/react-testing-library/setup#custom-render